### ISSUE TITLE
Localise variables

### ISSUE THREAD

**Issue by JonathanHandojo** [Autodesk-AutoCAD/AutoLispExt#166]
Is there an option to perform syntax checking like how we could set using the diagnostics of the general options in the Visual LISP Integrated Development Environment? It'll help a lot with localising variables for me when creating a command. If it's not possible, I can create my own function to debug this on the command line as well, but it'll be helpful.

**Comment by Sen-real** [comment]
@JonathanHandojo 
I don't think so; here're settings that we have right now. @nigma456 please correct me if I'm wrong.
![image](https://user-images.githubusercontent.com/57521938/133731939-e2de89dd-9d38-403f-a9c5-79a270fb3fd9.png)

**Comment by xg6636** [comment]
As the official editor (ide?),it's very incredible to lack the function of localizing variables.We need this function,fix it as soon as possible please.

**Comment by JDSComm** [comment]
I agree!
At least create an extension that could list the declared variables in each function, hopefully just the remaining global ones, to the output window.
To me it seems possible and not that difficult.
